.github/workflows/test-install-script.yml

```diff
@@ -60,53 +89,35 @@ jobs:
       with:
         fetch-depth: 0
 
-    - name: Install git and wget
+    - name: Install git and curl
       run: |
+        ${{ matrix.extra_setup_command }}
         ${{ matrix.package_manager }} update -y
         ${{ matrix.package_manager }} install -y git curl
+    - name: Install Python
+      run: |
+        ${{ matrix.package_manager }} install -y ${{ matrix.python_package }}
     - name: Install WasmEdge latest release
```

Comment:
We should have the latest release example installed in the system-wide path.

Reply:
Sorry, I am not following this. The python package name is different for different OSes. Hence I have did it in this way.

Reply:
Sorry, I mean that we should have `install.sh $HOME` and `install.sh /usr/` or `install.sh /usr/local` to test.